App.tsx

```diff
@@ -1,40 +1,24 @@
-import * as Font from "expo-font";
-import { StatusBar } from 'expo-status-bar';
 import React from 'react';
-import { SafeAreaProvider } from 'react-native-safe-area-context';
-
-import useCachedResources from './hooks/useCachedResources';
-import useColorScheme from './hooks/useColorScheme';
-import Navigation from './navigation';
+import SignIn from "./screens/SignIn";
+import PostScreen from "./screens/PostScreen"
+import { StyleSheet, View } from 'react-native';
 
 export default function App() {
-  const isLoadingComplete = useCachedResources();
-  const colorScheme = useColorScheme();
-
-
-  Font.loadAsync({
-    "Roboto-Black": require("./assets/fonts/Roboto-Black.ttf"),
-    "Roboto-BlackItalic": require("./assets/fonts/Roboto-BlackItalic.ttf"),
-    "Roboto-Bold": require("./assets/fonts/Roboto-Bold.ttf"),
-    "Roboto-BoldItalic": require("./assets/fonts/Roboto-BoldItalic.ttf"),
-    "Roboto-Italic": require("./assets/fonts/Roboto-Italic.ttf"),
-    "Roboto-Light": require("./assets/fonts/Roboto-Light.ttf"),
-    "Roboto-LightItalic": require("./assets/fonts/Roboto-LightItalic.ttf"),
-    "Roboto-Medium": require("./assets/fonts/Roboto-Medium.ttf"),
-    "Roboto-MediumItalic": require("./assets/fonts/Roboto-MediumItalic.ttf"),
-    "Roboto-Regular": require("./assets/fonts/Roboto-Regular.ttf"),
-    "Roboto-Thin": require("./assets/fonts/Roboto-Thin.ttf"),
-    "Roboto-ThinItalic": require("./assets/fonts/Roboto-ThinItalic.ttf")
-  });
-
-  if (!isLoadingComplete) {
-    return null;
-  } else {
     return (
-      <SafeAreaProvider>
-        <Navigation colorScheme={colorScheme} />
-        <StatusBar />
-      </SafeAreaProvider>
+      <View>
+        <PostScreen/>
```

Comment:
Should PostScreen be in App.tsx? If this is just for testing, it should be removed before merging in.